### ISSUE TITLE
missing access token ?

### ISSUE THREAD

**Issue by changchichung** [boojack/slash#10]
I start a  slash container by using my docker-compose.yml
```
version: '2'
services:
  slash:
    container_name: 'slash'
    image: 'stevenlgtm/slash:latest'
      #volumes:
      #- ./slash:/var/opt/slash
    restart: unless-stopped
    ports:
      - '5231:5231'
```
after running up the container , I can see the slash login page , but I find no default account/password info in README, so I choose up sign up a new account .
after I fill out the form and click apply , it just redirect to create your account page again.
and I saw some log in docker
```
{"time":"2023-07-24T08:23:09Z","method":"GET","uri":"/auth/signup?","status":200,"error":""}
{"time":"2023-07-24T08:23:10Z","method":"GET","uri":"/api/v1/workspace/profile","status":200,"error":""}
{"time":"2023-07-24T08:23:11Z","method":"GET","uri":"/api/v1/user/me","status":401,"error":"code=401, message=Missing access token"}
```

I'm wondering if there is any default account/password ? or somehow to solve the register new account problem ??

**Comment by quaintdev** [comment]
I have same issue. Neither I can sign up nor sign in.

**Comment by boojack** [comment]
Could you provide the browser and version you are using?

**Comment by quaintdev** [comment]
I am using arm64 version on RPi 4GB and using desktop version of Firefox.

I am using latest version with podman instead of Docker. I don't think last part would be an issue.



________________________________

**Comment by meitevau** [comment]
I am using Docker Version on UnRAID nAS.

Signup not working with Firefox but working with Brave / Vivaldi / Chromium Browser under Linux Mint.

Login is also not working with Firefox but working with Brave / Vivaldi / Chromium Browser under Linux Mint.

meitevau

**Comment by boojack** [comment]
Already fixed in the latest version v0.3.1

**Comment by boojack** [comment]
Why cannot? https://hub.docker.com/r/stevenlgtm/slash/tags

<img width="1262" alt="image" src="https://github.com/boojack/slash/assets/24653555/c7e198c7-585d-46f4-a2c5-3ab65e7e8b46">
